CONTRIBUTING.md

```diff
@@ -39,7 +39,18 @@ to run the example projects.
 
 ## Releasing
 
-## Release Checklist
+### Upgrading the vendored libraries
+
+To upgrade bugsnag-cocoa, prepare an updated version of bugsnag-cocoa, then copy
+the Source and iOS directories. Assuming bugsnag-cocoa is located in the same
+directory as bugsnag-react-native:
+
+```
+cp -r ../bugsnag-cocoa/Source/* cocoa/vendor/bugsnag-cocoa/Source
+cp -r ../bugsnag-cocoa/iOS/* cocoa/vendor/bugsnag-cocoa/iOS
+```
+
```

Comment:
The iOS xcodeproj will also need to be copied over (especially if we're adding/removing source files)

Reply:
Isn't that in `iOS/`?

Reply:
Yes - my bad!

Reply:
No worries, I had to check!